src/main/osd/osd_elements.c

```diff
@@ -945,10 +945,21 @@ static void osdBackgroundHorizonSidebars(osdElementParms_t *element)
 static void osdElementLinkQuality(osdElementParms_t *element)
 {
     uint16_t osdLinkQuality = 0;
-    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) { // 0-300
-        osdLinkQuality = rxGetLinkQuality()  / 3.41;
-        tfp_sprintf(element->buff, "%c%3d", SYM_LINK_QUALITY, osdLinkQuality);
-    } else { // 0-9
+    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF)
+    { // 0-300
+        osdLinkQuality = rxGetLinkQuality() / 3.41;
+        const uint8_t osdRfMode = rxGetRfMode();
+        if (osdLinkQuality > 99)
```

Comment:
This is more complex than it needs to be. Just have a single section for 2 digits as the CRSF link quality is supposed to range from 0-99.

Reply:
ok